src/corelibs/U2Core/src/util/TextUtils.cpp

```diff
@@ -239,4 +239,14 @@ bool TextUtils::isWhiteSpace(const QString &text, int charIndex) {
     return WHITES.testBit(bitIndex);
 }
 
+QString TextUtils::skip(const QBitArray &map, const QString &text) {
```

Comment:
`map` isn't `QMap`, so, I think, it's better to rename this variable

Reply:
map here is for 'bit-map'.
I agree that we can find a better name for it, but 10+ other methods in this class use 'map' for bit-maps.